adam-core/src/test/scala/org/bdgenomics/adam/converters/FastqConverterSuite.scala

```diff
@@ -0,0 +1,142 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bdgenomics.adam.converters
+
+import htsjdk.samtools.ValidationStringency
+import org.apache.hadoop.io.Text
+import org.scalatest.FunSuite
+
+/**
+ * Created by zyxue on 2016-09-27.
+ */
+class FastqConverterSuite extends FunSuite {
```

Comment:
OK, you're right. I realized it's

`FastaConverter.scala` & `FastaConverterSuite`

so it should be 

`FastqRecordConverter` & `FastqRecordConverterSuite`